### DISCUSSION TITLE
[Adonis V5] File Storage in v5?

### DISCUSSION THREAD

**Discussion by aayush123** [adonisjs/core#1682] [Help]
Hey guys,

I'm starting a new project in Adonis v5 and was wondering how to handle file storage in v5.
v4 has a nice guide and a package for file storage.
Does the same apply to v5, and will a guide be added to v5's documentation?

Thanks!

**Comment by fairhat** [comment]
From the Docs:
https://preview.adonisjs.com/guides/http/file-uploads

**Reply by aayush123** [reply]
I've gone through this. However, this does not mention how to handle streaming the file to s3, although it is mentioned in the introduction.
Moreover, it does not specify how exactly to pull a package to handle to handle file storage (referring to other guides where it's stated that we need to pull from alpha branch)

**Reply by fairhat** [reply]
https://github.com/adonisjs/core/discussions/1545#discussioncomment-60419

Seems like people wait for the blog post

**Reply by aayush123** [reply]
Yeah, thanks for the link. Guess I'll get in the queue too 😅
